00-RELEASENOTES

```diff
@@ -9,6 +9,36 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Valkey 8.0.3  -  Released Wed 23 Apr 2025
+================================================================================
+
+Upgrade urgency SECURITY: This release includes security fixes we recommend you
+apply as soon as possible.
+
+Bug fixes
+=========
+* Optimize RDB load performance and fix cluster mode resizing on replica side (#1199)
+* Fix memory leak in forgotten node ping ext code path (#1574) 
+* Fix cluster info sent stats for message with light header (#1563)
+* Use fake client flag to replace not conn check (#1198)
+* Fix module LatencyAddSample still work when latency-monitor-threshold is 0 (#1541)
+* Fix raxRemove crash at memcpy() due to key size exceeds max Rax size (#1722)
```

Comment:
This PR title wasn't very good as a release notes title IMHO.

"Fix crash" sounds severe, but this one is actually just a hypothetical crash. It was never reported as a bug. I think it was just discovered by reading code or by fuzzing.

* It doesn't affect streams because streams don't use long key names.
* ACL? Huge key names are unlikely there too.
* Client tracking with huge keys > 512MB? Maybe..
* The module API for "Dict"? It's implemented as a rax. But then we could have reproduced it with a module API test case. This fix just has a unit test of rax.

Maybe we can soften it saying "potential crash"?

```suggestion
* Fix potential crash in radix tree recompression of huge keys (#1722)
```

Reply:
Sounds good to me. Applied